orchestration/src/main/java/com/sap/ai/sdk/orchestration/UserMessage.java

```diff
@@ -93,6 +93,12 @@ public UserMessage withImage(@Nonnull final String imageUrl) {
   public ChatMessage createChatMessage() {
     final var contentList = new LinkedList<UserChatMessageContentItem>();
 
+    if (this.content().items().size() == 1
```

Comment:
(Minor)

Inconsistent `this.content()` (here) vs `content` (in `AssistantMessage`)